grpc-sys/build.rs

```diff
@@ -309,18 +309,23 @@ fn bindgen_grpc(mut config: bindgen::Builder, file_path: &PathBuf) {
         .whitelist_type(r"\bgrpcwrap_.*")
         .whitelist_type(r"\bcensus_context.*")
         .whitelist_type(r"\bverify_peer_options.*")
+        .whitelist_type(r"(__)?uint8_t")
+        .whitelist_type(r"(__darwin_)?size_t")
+        .whitelist_type(r"(__)?uint32_t")
+        .whitelist_type(r"(__)?int64_t")
+        .whitelist_type(r"(__)?int32_t")
         .blacklist_function(r"\bgpr_mu_.*")
         .blacklist_function(r"\bgpr_cv_.*")
         .blacklist_function(r"\bgpr_once_.*")
         .blacklist_type(r"gpr_mu")
         .blacklist_type(r"gpr_cv")
         .blacklist_type(r"gpr_once")
         .constified_enum_module(r"grpc_status_code")
-        .no_copy("grpc_slice")
         .default_enum_style(bindgen::EnumVariation::Rust {
             non_exhaustive: false,
-        })
-        .generate()
+        });
+    println!("running {}", cfg.command_line_flags().join(" "));
```

Comment:
Is this necessary? I think `cargo build --vv` should output something similar.

Reply:
Tried and couldn't find it from stdout, stderr, $OUT_DIR/../output, $OUT_DIR/../stderr.